inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/MonitoringPage.java

```diff
@@ -0,0 +1,524 @@
+        Label name = new Label("name",currentProject.getObject().getName());
+        add(name);
+
+
+        //Headers of the table
+        List<String> headers = new ArrayList<>();
+        headers.add(getString("Document"));
+        headers.add(getString("Finished"));
+        headers.add(getString("Processed"));
+        headers.add(getString("Metadata"));
+
+        //Data Provider for the table
+        DataProvider dataProvider = new DataProvider(documentService.
+            listSourceDocuments(currentProject.getObject()),
+            headers, documentService.listAnnotationDocuments
+            (currentProject.getObject()));
+
+        //Init defaultDocumentsNumberTextField
+        NumberTextField defaultNumberDocumentsTextField =
+            new NumberTextField("defaultDocumentsNumberTextField", Integer.class);
+
+        //Set minimum value for input
+        defaultNumberDocumentsTextField.setMinimum(1);
+        //After upstream change
+        defaultNumberDocumentsTextField.setRequired(true);
+
+        ModalWindow modalWindow = new ModalWindow("modalWindow");
+        workload.add(modalWindow);
+
+
+
+
+        //Get value for the project and set it accordingly
+        //TODO get correct value after upstream change
+        defaultNumberDocumentsTextField.setDefaultModel(new CompoundPropertyModel<Integer>(6));
+        defaultNumberDocumentsTextField.setConvertEmptyInputStringToNull(false);
+
+        //add AJAX event handler on changing input value
+        defaultNumberDocumentsTextField.add(new OnChangeAjaxBehavior()
+        {
+            private static final long serialVersionUID = 2607214157084529408L;
+
+            @Override
+            protected void onUpdate(AjaxRequestTarget ajaxRequestTarget)
+            {
+                defaultAnnotations.setObject(Integer.parseInt
+                    (defaultNumberDocumentsTextField.getInput()));
+            }
+        });
+
+        workload.add(defaultNumberDocumentsTextField);
+
+        //Columns of the table
+        List<IColumn> columns = new ArrayList<>();
+
+        //Filter properties
+        FilterForm<Filter> filter = new FilterForm<>(getString("filter"), dataProvider);
+        filter.setOutputMarkupId(true);
+
+        //Filter Textfields and their AJAX events
+        TextField<String> userFilterTextField = new TextField("userFilter",
+            PropertyModel.of(dataProvider, "filter.username"), String.class);
+
+        userFilterTextField.add(new AjaxFormComponentUpdatingBehavior("onchange") {
+            @Override
+            protected void onUpdate(AjaxRequestTarget ajaxRequestTarget) {
+                ajaxRequestTarget.add(workload);
+            }
+        });
+
+        TextField<String> documentFilterTextField = new TextField("documentFilter",
+                PropertyModel.of(dataProvider, "filter.documentName"), String.class);
+
+        documentFilterTextField.add(new AjaxFormComponentUpdatingBehavior("onchange") {
```

Comment:
Use our `LambdaAjaxFormComponentUpdatingBehavior` since it doesn't require the need to use inner classes and make the code less verbose.